### ISSUE TITLE
OpenCV local feature orientation is oppositte 

### ISSUE THREAD

**Issue by ducha-aiki** [kornia/kornia#573]
Now to extract the same patches as OpenCV one needs to convert by multiplication by -1:

```python
from kornia.feature import laf_from_center_scale_ori as get_laf
def laf_from_opencv_kpts(kpts, mrSize=6.0, device=torch.device('cpu')):
    N = len(kpts)
    xy = torch.tensor([(x.pt[0], x.pt[1]) for x in kpts ], device=device, dtype=torch.float).view(1, N, 2)
    scales = torch.tensor([(mrSize * x.size) for x in kpts ], device=device, dtype=torch.float).view(1, N, 1, 1)
    angles = torch.tensor([(x.angle) for x in kpts ], device=device, dtype=torch.float).view(1, N, 1)
    laf = get_laf(xy, scales, -angles)
    return laf

```

ToDo: make the same direction, as in OpenCV.

**Comment by connorlee77** [comment]
is this function already in Kornia?

**Comment by connorlee77** [comment]
@ducha-aiki Question: If I convert a set of detected OpenCV keypoints into lafs, would resulting patches retrieved from **extract_patches_from_pyramid** be the same as the patches extracted from the OpenCV gaussian pyramid? I ask because I noticed differences in the implementation of the scale pyramid and OpenCV's gaussian pyramid.

**Comment by ducha-aiki** [comment]
> is this function already in Kornia?

No, and will not be, because it depends on OpenCV. Although it would be good to put this code snippet into the documentation.
Regarding patches and pyramid - OpenCV do not have "extract_patches" function. The implementation is almost identical to extract_patches
https://github.com/ducha-aiki/extract_patches/blob/master/nbs/core.ipynb

**Comment by edgarriba** [comment]
@ducha-aiki what should we do here ?

**Comment by ducha-aiki** [comment]
@edgarriba I will do PR soon
